### PR TITLE
Update English wording

### PR DESCRIPTION
**Pull request by dg4101**
## 📝 関連issue / Related Issues

 総合issue #680

## ⛏ 変更内容 / Details of Changes

- 英語の表記変更
citizens -> residents (住民という意味ではこちらの自然と思われます)
Positive patient -> Confirmed patient　(ほか2項目と表現を統一)
Number of tested　-> Number of tests conducted
Number of reports -> Number of inquiries

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -149,10 +149,10 @@
...
     "検査陽性者の状況": "Details of confirmed cases",
     "陽性患者数": "Number of confirmed cases",
-    "陽性患者の属性": "Positive patient attributes",
-    "検査実施数": "Number of tested",
-    "新型コロナコールセンター相談件数": "Number of reports to COVID-19 telephone advisory center",
-    "新型コロナ受診相談窓口相談件数": "Number of reports to combined telephone advisory center",
+    "陽性患者の属性": "Confirmed patient attributes",
+    "検査実施数": "Number of tests conducted",
+    "新型コロナコールセンター相談件数": "Number of inquiries to COVID-19 telephone advisory center",
+    "新型コロナ受診相談窓口相談件数": "Number of inquiries to combined telephone advisory center",
```
Comment (PichuChen) [inline pages/index.vue]: Please see https://docs.google.com/spreadsheets/d/1aksSfWIXE_L3rFYFjHz9xtVhh0n7ealMQ0eEz8GGPlw/edit#gid=0 Row 27 and 152, there are new wording.

Comment (dg4101) [reply pages/index.vue]: thank you. I added comment there.

**Comment by MaySoMusician** [comment]
Could you change the base branch from `tokyo-metropolitan-gov:dev-i18n` to `tokyo-metropolitan-gov:development` ?

**Comment by dg4101** [comment]
base branch updated to `tokyo-metropolitan-gov:development`

**Comment by MaySoMusician** [comment]
@dg4101 And it seems that `SideNavigation.vue` has some conflicts, could you resolve this?

**Comment by dg4101** [comment]
I updated that as well. Could you check?

**Comment by halsk** [comment]
Well, somehow deploying was failed. could you please check it?

**Comment by mikkame** [comment]
deploy failed.
retry now.

**Pull request merged**